src/main/java/xyz/aadev/generitech/common/tileentities/power/TileEntityPower.java

```diff
@@ -87,16 +108,17 @@ public void transferPower(){
         World worldIn = getWorld();
         long inputba = TeslaUtils.distributePowerToAllFaces(worldIn,pos,T0transfer,true);
         long test = inputba / T0transfer;
-        if (inputba!=0 && container.getStoredPower() > test){
+        if (test==0)test=1;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS00122)